starter-code/css/main.css

```diff
@@ -6,3 +6,148 @@ Green: #00B172
 White: #FFF
 
 */
+*{
+  margin: 0;
+  padding: 0;
+}
+
+body{
+  position:absolute;
+}
+
+header{
+  display:flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: whitesmoke;
+  border-bottom: #1A1A1A 0.5px;
+  position: fixed;
+}
+
+#logo{
+  display:inline;
+  width: 15%;
+  height: 15%;
+  padding: 1vh 0;
+}
+
+ .col-3 img{
+  display: inline;
+  width: 60%;
+}
+
+.col-3{
+  margin:10% 0;
+}
+
+ul {
+  display: flex;  
+}
+
+li{
+  margin:20px;
+  list-style: none;
+}
+
+li a{
+  text-decoration: none;
+  color: #1A1A1A;
+}
+
+
+
+.bg{
+  background-image: url(/starter-code/images/landing.jpg);
```

Comment:
you have to use a relative path here : background-image: url(../images/landing.jpg);